docs/usage-with-routing.stories.mdx

```diff
@@ -44,6 +45,22 @@ const CarbonLinkWithReactRouter = ({ to, children }) => {
   return <Link onClick={onClick}>{children}</Link>;
 };
 
+const CarbonTabsWithReactRouter = ({ children }) => {
+  const history = useHistory();
+  const handleTabChange = useCallback(
+    (tabId) => {
+      history.push(tabId);
+    },
+    [history]
+  );
+
+  return (
+    <Tabs history={history} onTabChange={handleTabChange}>
```

Comment:
this will cause `history.push` to be called twice based on the code in `tabs.component.js`, I think you also need to set `setLocation=false`.